hazelcast/src/main/java/com/hazelcast/internal/diagnostics/OverloadedConnectionsPlugin.java

```diff
@@ -198,13 +199,11 @@ private void clear() {
     /**
      * Samples the queue.
      *
-     * @param q the queue to sample.
-     * @return the number of samples. If there were not sufficient samples, -1 is returned.
+     * @param q the queue to sample
+     * @return the number of samples (if there were not sufficient samples, -1 is returned)
      */
     private int sample(Queue<OutboundFrame> q) {
-        for (OutboundFrame frame : q) {
-            packets.add(frame);
-        }
+        packets.addAll(q);
```

Comment:
Was a suggestions from IDEA, which I already followed in the `EventQueuePlugin`.